README.markdown

```diff
@@ -131,6 +131,12 @@ An array of services to start, normally `puppet` and `mcollective`. If the array
 
 Alternate source from which you wish to download the latest version of Puppet.
 
+####`install_dir`
+
+The directory the puppet agent should be installed to. This is only applicable for windows operating systems.
+This only applies to an agent upgrade and will not change an already installed agent.
```

Comment:
Yes, thanks. I'm going to tweak the wording slightly to `This only applies when upgrading the agent to a new version; it will not cause re-installation of the same version to a new location.`